### ISSUE TITLE
Display elements bounding boxes

### ISSUE THREAD

**Issue by Fritz-the-fox** [sketchplugins/plugin-requests#45]
Hey! 
Some abandoned plugins used to do it rather well (https://github.com/pravdomil/Sketch-Show-Bounding-Boxes) I know that lots of designers — including myself — really appreciate having all bounding boxes on screen to gain precision in order to be pixel perfect. Well I'de be happy to have a built-in solution or a plugin.

**Comment by himonoye** [comment]
Yes Please! It should be a native function as I don't think it could be achieved by the current Sketch API
